modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -70,7 +70,26 @@ The following table compares the Object Store REST API operations to Object Stor
 . `objectStore.store(key, value);`
 . `objectStore.store(value, key);`
 
-== About the Object Store v2 APIs Workflow
+== Object Store V2 Data Center Locations
+
+The following REST domains can be used for the application locations with Object Store:
+
+[source,xml,linenums]
+----
+object-store-us-east-2.anypoint.mulesoft.com
```

Comment:
I think it would help if you categorize this based on control plane. so for prod control plane, you will have 

https://object-store-us-east-1.anypoint.mulesoft.com/api/v1/ping
https://object-store-us-east-2.anypoint.mulesoft.com/api/v1/ping
https://object-store-us-west-2.anypoint.mulesoft.com/api/v1/ping
https://object-store-ap-southeast-1.anypoint.mulesoft.com/api/v1/ping
https://object-store-ap-southeast-2.anypoint.mulesoft.com/api/v1/ping
https://object-store-ap-northeast-1.anypoint.mulesoft.com/api/v1/ping
https://object-store-ca-central-1.anypoint.mulesoft.com/api/v1/ping
https://object-store-eu-central-1.anypoint.mulesoft.com/api/v1/ping
https://object-store-eu-west-1.anypoint.mulesoft.com/api/v1/ping
https://object-store-eu-west-2.anypoint.mulesoft.com/api/v1/ping
https://object-store-sa-east-1.anypoint.mulesoft.com/api/v1/ping


for prod-eu control plane 

https://object-store-eu-central-1.eu1.anypoint.mulesoft.com/api/v1/ping
https://object-store-eu-west-1.eu1.anypoint.mulesoft.com/api/v1/ping

Reply:
you can remove the /api/v1/ping too. but my question is, why do we want to add all the DNS like this? I haven't seen this in other service docs.

Reply:
So, eu-west-1 and eu-west-2 are not in prod-eu?

Reply:
I've asked @robday-sf to clarify the requirements.

Reply:
Rob slack-approved the changes.